### ISSUE TITLE
Suggested rooms are no longer

### ISSUE THREAD

**Issue by kittykat** [vector-im/element-android#5715]
### Steps to reproduce

1. Go to Explore rooms for a space which has at least one suggested room

#### What did you expect?
Suggested rooms are at the top of the list and clearly marked as suggested

#### What happened instead?
Can't see which rooms are suggested because they're not differentiated from rooms which aren't suggested

### Operating system version

Android 11

### Application version and app store

1.4.8 [40104082] (G-b7130) from app store

### Will you send logs?

No

**Comment by niquewoodhouse** [comment]
Not sure if I'm right but I think they used to have 'suggested' as a label to the right of the room name?

It might be a bit poor for smaller phones, and when rooms have long names, or its in a language with many characters....

Maybe something like this is better:

<img width="408" alt="Room directory@2x" src="https://user-images.githubusercontent.com/1587166/162421817-c4a36c86-27a3-4140-ba10-04cb47f81f18.png">

- Put suggested in a section at top
  - Issues: if user is a member of a suggested thing, should they stay in the suggested section?  I think they probably should but this would be opposite to the rooms tab when you join something suggested (it disappears from the suggested rooms section and joins the room list underneath)
- All other rooms underneath

**Comment by niquewoodhouse** [comment]
@kittykat do you remember seeing this in Android in exploring?

**Comment by kittykat** [comment]
I'm pretty sure there used to be a label to the right. I'd be concerned with screen height becoming an issue in the above case. Also I think we might not have the full list of suggested rooms until the user scrolls because of pagination - @fedrunov can you confirm?

**Comment by niquewoodhouse** [comment]
> I'm pretty sure there used to be a label to the right. I'd be concerned with screen height becoming an issue in the above case. Also I think we might not have the full list of suggested rooms until the user scrolls because of pagination - @fedrunov can you confirm?

I thought it might have been there but I don't understand how that would fit in with the 'Join' 'Open' buttons

**Comment by kittykat** [comment]
> I thought it might have been there but I don't understand how that would fit in with the 'Join' 'Open' buttons

I think it was in a grey, smaller text after the room name, similar to how we have the MXID after the DM name on web in the new search

**Comment by niquewoodhouse** [comment]
@kittykat Like this?

<img width="556" alt="image" src="https://user-images.githubusercontent.com/1587166/164221711-7e8b0e2f-5718-465e-806a-fb0b3bceabb6.png">

**Comment by kittykat** [comment]
> @kittykat Like this?
> 
> <img alt="image" width="556" src="https://user-images.githubusercontent.com/1587166/164221711-7e8b0e2f-5718-465e-806a-fb0b3bceabb6.png">

Yes!

**Comment by niquewoodhouse** [comment]
Thanks @kittykat 

Based on the above @ericdecanini we'd like to add suggested as microcopy next to the room name.  Here are some options below. Is there anything simplest to achieve, as of course we're dealing with dynamic content (where a title can be any number of characters):

<img width="620" alt="image" src="https://user-images.githubusercontent.com/1587166/164232793-0dd3d6a0-d777-4197-bdd5-09e321f533cc.png">

**Comment by ericdecanini** [comment]
@niquewoodhouse the simplest way out of these would be for the Suggested tag to be below the room name, irregardless of the length. Having it either on the side or below like it is above is possible but it would be far simpler to opt for a single consistent placement.

Also @kittykat I'm a little confused with the regression tag. Is this something that was present in an old version but lost in recent versions?

**Comment by kittykat** [comment]
@ericdecanini yes, suggested rooms did used to be marked some time ago. I'm not sure when we lost the label but it was probably some time this year.

**Comment by ericdecanini** [comment]
cc @kittykat @fedrunov since you guys are most likely to know the answers to these

I went back to a release from November and didn't see the suggested tag, but I'm unsure whether that's because the regression exists in that version or the space I'm looking in (Element Corp) doesn't have suggested rooms.

I certainly don't see suggested rooms in the same space on Element Web. Is there a space we know where there's suggested rooms where I can test and confirm this existing on an old version? It would be much better for me to do so before approaching this with a fix

**Comment by niquewoodhouse** [comment]
Looking back through older designs, I can see the design for this view maybe didn't have the Open and Join buttons, which made more space for the suggested label on the right.  Not sure when the Open and Join buttons were added here.

@ericdecanini happy to go ahead with the single consistent placement below for now.

I think longer term having a suggested section on top might be optimal for people looking to explore a space.  Even if we can't show all the suggested things, we could have a view all button that then showed everything suggested.

**Comment by ericdecanini** [comment]
@niquewoodhouse hope it's alright that I added you as a reviewer on the above PR for design review

**Comment by kittykat** [comment]
> I certainly don't see suggested rooms in the same space on Element Web. Is there a space we know where there's suggested rooms where I can test and confirm this existing on an old version? It would be much better for me to do so before approaching this with a fix

I could have been mistaken, as I do test both on iOS and Android, but we the test case for suggested rooms marked as passing in 1.3.4 with some issues where the rooms don't show up :shrug:
